src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/OperatorWrapCheck.java

```diff
@@ -104,6 +135,65 @@
  *   &lt;property name="option" value="eol"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public static void main(String[] args) {
+ *             int b
+ *                     = 10; // violation, '=' should be on previous line
+ *             int c =
+ *                     10; // ok
+ *             b
+ *                     += 10; // violation, '+=' should be on previous line
+ *             b +=
+ *                     10; // ok
+ *             c
+ *                     *= 10; // violation, '*=' should be on previous line
+ *             c *=
+ *                     10; // ok
+ *             c
+ *                     -= 5; // violation, '-=' should be on previous line
+ *             c -=
+ *                     5; // ok
+ *             c
+ *                     /= 2; // violation, '/=' should be on previous line
+ *             c /=
+ *                     2; // ok
+ *             c
+ *                     %= 1; // violation, '%=' should be on previous line
+ *             c %=
+ *                     1; // ok
+ *             c
+ *                     &gt;&gt;= 1; // violation, '&gt;&gt;=' should be on previous line
+ *             c &gt;&gt;=
+ *                     1; // ok
+ *             c
+ *                 &gt;&gt;&gt;= 1; // violation, '&gt;&gt;&gt;=' should be on previous line
+ *             c &gt;&gt;&gt;=
+ *                     1; // ok
```

Comment:
there is no need for so many examples, 2-3 is enough

Reply:
@strkkk  we need to show users case of every tokens which are there in the congif this is the reason i showed all the example as i did in #7831

Reply:
As for me, there is too much examples.
Ok, lets remove ok code examples and keep only 2 or 3 at the top.

Reply:
this item is not corrected

Reply:
What do u mean by it is not corrected @strkkk ?

Reply:
I'll keep 4 Ok message code and remove rest of them is it fine @strkkk ?

Reply:
@harsh-kukreja yes it is ok